File: 2004/j2.cpp
// code by savir singh

#include <iostream>
#include <string>
#include <vector>
using namespace std;

int main()
{
    int year;
    int limit;
    vector<int> years;
    cin >> year;
    cin >> limit;
    int year1 = year;

    while (true) {
        year += 60;
        if (year<=limit) {
            years.push_back(year);
        }
        else {
            break;
        }
    }

    cout << "All positions change in year " + to_string(year1) << endl;

    for (int i : years) {
        cout << "All positions change in year " + to_string(i) << endl;
    }
}

File: 2004/j2.py
# code by savir singh

year = int(input())
year1 = year
limit = int(input())
years = []

while True:
    year += 60
    if year <= limit:
        years.append(year)
    else:
        break

print("All positions change in year " + str(year1))

for y in years:
    print("All positions change in year " + str(y))
